toc/working-groups/vulnerability-management.md

```diff
@@ -0,0 +1,40 @@
+# Vulnerability Management: Working Group Charter
+
+## Mission
+
+Provides discreet management of security vulnerabilities issues relevant for active CF projects.
+
+## Goals
+
+* Provide a single point of contact for security vulnerability reporting and management.
+* Provide management of security vulnerability reports through to resolution, including but not limited to triage, reporter and team coordination, embargo negotiation, CVSS scoring, CVE assignments, pre-disclosure and disclosure.
+
+## Scope
+
+* Triage incoming security vulnerability reports to [security@cloudfoundry.org](mailto:security@cloudfoundry.org).
+* Manage vulnerabilities through dedicated slack channels.
+* When appropriate, negotiate suitable embargo periods with the reporter to afford component teams time to fix the issue before it becomes known publicly.
+* When appropriate, assign CVE numbers to vulnerabilities/fixes.
+* Publish pre-disclosures to allow all CF distributions time to adopt fixes for high/critical vulnerabilities before they become known publicly.
+* Publish disclosures of reported security vulnerabilities.
+
+## Non-Goals
+
+* Add security-related features to Cloud Foundry projects.
+
+## Technical Lead(s): 
+- [Paul Warren](https://github.com/paulcwarren)
```

Comment:
I think @paulcwarren is trying to get out of this role he's been playing. Do we have other ideas in mind for who could do this in the OSS community?

Reply:
I don't know if he's found anybody as yet.

Reply:
Sure.  Go ahead Lee.  For now, its me so merging is fine.